magpie/ui/management/templates/edit_group.mako

```diff
@@ -4,138 +4,215 @@
+                            %if group_name not in MAGPIE_FIXED_GROUP_EDITS:
+                            <input type="submit" value="Edit" name="edit_group_name" class="button theme">
+                            %endif
+                            </label>
+                        %endif
+                    </p>
+                </form>
+                <form id="edit_description" action="${request.path}" method="post">
+                    <p class="panel-line">
+                        <span class="panel-entry">Description: </span>
+                        %if edit_mode == "edit_description" and group_name not in MAGPIE_FIXED_GROUP_EDITS:
+                            <label>
+                            <input type="text" placeholder="description" name="new_description"
+                                   id="input_description" onkeyup="adjustWidth('input_description')"
+                                %if description:
+                                    value="${description}"
+                                %endif
+                            >
+                            <input type="submit" value="Save" name="save_description" class="button theme">
+                            <input type="submit" value="Cancel" name="no_edit" class="button cancel">
+                            </label>
                         %else:
-                            <span class="panel_value">${group_name}</span>
-                            <input type="submit" value="Edit" name="edit_group_name">
+                            <label>
+                            <span class="panel-value">
+                                %if description:
+                                    ${description}
+                                %else:
+                                    n/a
+                                %endif
+                            </span>
+                            %if group_name not in MAGPIE_FIXED_GROUP_EDITS:
+                            <input type="submit" value="Edit" name="edit_description" class="button theme">
+                            %endif
+                            </label>
                         %endif
                     </p>
-                </div>
+                </form>
+                <form id="edit_discoverable" action="${request.path}" method="post">
+                    <p class="panel-line panel-line-checkbox">
+                        <span class="panel-entry">Discoverable: </span>
+                        <label>
+                        <input type="checkbox" name="new_discoverable" id="input_discoverable"
+                            %if discoverable:
+                               checked
+                            %endif
+                            %if group_name in MAGPIE_FIXED_GROUP_EDITS:
+                               disabled
+                            %else:
+                               onchange="document.getElementById('edit_discoverable').submit()"
```

Comment:
Try to make a group discoverable [worked]. Then uncheck the box and got : 
400 Bad Request
Invalid POST request for group edit.
Is it a one way operation?

Reply:
Should be 2-way.
Probably the checkbox value is not correctly retrieved, (eg `None` instead of `False` for `discoverable`). 
Will look into it.

Reply:
If you want to check the error I got (and for the error.mako template) you can check the group tb16 on our ogc-ades instance. I make it discoverable and it's stuck in this state.